backend/src/main/java/atlantbh/restaurants/models/BaseModel.java

```diff
@@ -0,0 +1,24 @@
+package atlantbh.restaurants.models;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+import javax.validation.constraints.NotNull;
+
+@MappedSuperclass
+public abstract class BaseModel {
+
+    private long id;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @NotNull(message = "User ID cannot be null")
```

Comment:
This is not just user model, so validation message is not correct.

Reply:
Fixed c2ed9b4060983e1312e68ee9cdd1de2be1986207